### ISSUE TITLE
Load Named State issue (character no longer support in BizHawk 2.6.3)

### ISSUE THREAD

**Issue by Bif-u** [TASVideos/BizHawk#2954]
### Summary
[//]: # "Briefly describe what's broken. Include relevant details: loaded core, loaded rom's hash, open tools, running scripts... You can embed a screenshot if it's easier to show the bug, but if you need more than one please put them at the end."
I wanted to load a BizHawk 2.6.1 savestate with BizHawk 2.6.3 and the emulator crashed.
There is an `é` in this savestate filename, and without this character it works.

### Output
[//]: # "Paste the contents of the error dialog if there is one (try Ctrl+C, it usually works), or paste the output from the Lua Console, or delete this section."
```
System.Collections.Generic.KeyNotFoundException: La clé donnée était absente du dictionnaire.
   à System.ThrowHelper.ThrowKeyNotFoundException()
   à System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   à BizHawk.Bizware.BizwareGL.StringRenderer.Measure(String str)
   à BizHawk.Client.Common.OSDManager.GetCoordinates(IBlitter g, MessagePosition position, String message)
   à BizHawk.Client.Common.OSDManager.DrawMessage(IBlitter g, UIMessage message, Int32 yOffset)
   à BizHawk.Client.Common.OSDManager.DrawMessages(IBlitter g)
   à BizHawk.Client.Common.DisplayManagerBase.<>c__DisplayClass43_0.<BuildDefaultChain>b__0()
   à BizHawk.Client.Common.DisplayManagerBase.RunFilterChainSteps(Int32& rtCounter, RenderTarget& rtCurr, Boolean& inFinalTarget)
   à BizHawk.Client.EmuHawk.DisplayManager.UpdateSourceDrawingWork(JobInfo job)
   à BizHawk.Client.Common.DisplayManagerBase.UpdateSourceInternal(JobInfo job)
   à BizHawk.Client.EmuHawk.MainForm.Render()
   à BizHawk.Client.EmuHawk.MainForm.ProgramRunLoop()
   à BizHawk.Client.EmuHawk.Program.SubMain(String[] args)
```

**Comment by YoshiRulz** [comment]
I have no idea how that could happen, but I've had a go at fixing it: https://github.com/TASVideos/BizHawk/commit/56de8590426761b1a1a7d54b937a4d4e6b290f7f. Please try [this dev build](https://gitlab.com/TASVideos/BizHawk/-/pipelines/383116287/builds).

(Note that we don't support loading savestates from older versions. If that fix does work and the savestate is loaded, it may corrupt the game. It depends on which core you're using.)

**Comment by Bif-u** [comment]
Thank you!

(I forgot to mention that it was a PSX game, and that I tried to use `Save Named State` directly from BizHawk 2.6.3 with an `é` in the filename and it crashed with the same outpout and worked without an `é`.)

**Comment by YoshiRulz** [comment]
So is the dev build still broken or not?

**Comment by TizianoMattei** [comment]
Yup @YoshiRulz  i repro'ed on 2.6.3 and then tested on dev build.

Output on 2.6.3: 
System.Collections.Generic.KeyNotFoundException: La chiave specificata non era presente nel dizionario.
   in System.ThrowHelper.ThrowKeyNotFoundException()
   in System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   in BizHawk.Bizware.BizwareGL.StringRenderer.Measure(String str)
   in BizHawk.Client.Common.OSDManager.GetCoordinates(IBlitter g, MessagePosition position, String message)
   in BizHawk.Client.Common.OSDManager.DrawMessage(IBlitter g, UIMessage message, Int32 yOffset)
   in BizHawk.Client.Common.OSDManager.DrawMessages(IBlitter g)
   in BizHawk.Client.Common.DisplayManagerBase.<>c__DisplayClass43_0.<BuildDefaultChain>b__0()
   in BizHawk.Client.Common.DisplayManagerBase.RunFilterChainSteps(Int32& rtCounter, RenderTarget& rtCurr, Boolean& inFinalTarget)
   in BizHawk.Client.EmuHawk.DisplayManager.UpdateSourceDrawingWork(JobInfo job)
   in BizHawk.Client.Common.DisplayManagerBase.UpdateSourceInternal(JobInfo job)
   in BizHawk.Client.EmuHawk.MainForm.Render()
   in BizHawk.Client.EmuHawk.MainForm.ProgramRunLoop()
   in BizHawk.Client.EmuHawk.Program.SubMain(String[] args)
